pkgs/development/python-modules/beancount_docverif/default.nix

```
@@ -0,0 +1,46 @@
+{ stdenv, buildPythonPackage, fetchPypi, isPy3k
```

personal preference:
```suggestion
{ lib, buildPythonPackage, fetchPypi, isPy3k
```